eva_sub_cli/report.py

```diff
@@ -17,12 +17,16 @@ def get_logo_data():
 
 def generate_report(validation_results, validation_date, submission_dir, vcf_fasta_analysis_mapping, project_title,
                     consent_statement_required, subdir, template_file):
+
+    from eva_sub_cli.validators.validator import RUN_STATUS_KEY
```

Comment:
should go at the top of the file

Reply:
It is complaining about circular dependency if I put on top of the file, since we import from the report.py in validator.py as well